src/SDKs/Compute/Management.Compute/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("15.0.0.0")]
-[assembly: AssemblyFileVersion("15.0.0.0")]
+[assembly: AssemblyVersion("16.0.0.0")]
```

Comment:
@shahabhijeet   It should be 16.1.   I updated this PR.  I don't know why the AssemblyVersion was not updated last time.  It looks like a mistake last time.